forslag.md

```diff
@@ -1,68 +1,60 @@
-Lige nu er store dele af vores digitale liv ejet af få private virksomheder. Derfor er vi som land dybt afhængige af dem &ndash; og det er dyrt og inffektivt!
+Lige nu er store dele af vores digitale liv ejet af få private virksomheder. Derfor er vi som land dybt afhængige af dem &ndash; og det er dyrt og ineffektivt!
 Lad os sammen skabe en fremtid hvor Danmark går forrest med billig, moderne og innovativ IT.
 
 ### Offentlig IT er billig IT
 Når staten køber et IT system i dag køber de katten i sækken.
 Staten ejer ikke engang systemet.
 Det gør den virksomhed, der har bygget det.
-Tænk hvis det også gjaldt for din bil: så ville producenten svejse motorhjelmen fast så snart den blev solgt.
-Selv ikke din mekaniker måtte få lov til at kigge i den.
-Så hvis bilen gik i stykker, ville du være tvunget til at køre den ødelagte bil hjem Asien, hvor den blev bygget.
+Tænk hvis det også gjaldt for din bil: Så ville producenten gøre teknikken i bilen utilgængelig for selv din mekaniker, og du ville derfor være helt afgængig af producenten til at reparere bilen igen.
 
 Desværre er det præcis det, der sker med vores offentlige IT.
-Alene i årene 2000-2014 har staten Danmark kastet 2,902 milliarder kroner væk på skrottet IT.<sup><a name="dr8">2</a></sup>
-Det svarer til 585 fuldtidsansatte sygeplejersker i de 14 år.
+Alene i årene 2000-2014 har staten Danmark brugt 2902 millioner kroner på skrottet IT.<sup><a name="dr8">2</a></sup>
+Det svarer til 585 fuldtidsansatte sygeplejersker i 14 år.
 
 Hvis systemerne havde været ejet af staten kunne vi &ndash; ligesom med en ødelagt bilmotor &ndash; redde og genbruge store dele af dem, selv efter de var brudt sammen.
 
-Det er en langt bedre løsning på en bedre digital hverdag i Danmark, fordi det vil gøre os til herre over eget hus. Vi skal kræve
+Det er en langt bedre løsning på en bedre digital hverdag i Danmark, og vi skal derfor kræve
 at vores leverandører lever op til åbne standarder så vi kan bygge videre på vores infrastruktur.
 
 ### Åbenhed giver kvalitet
 Millioner af sygdomsforløb, straffeattester og andre fortrolige
-dokumenter bliver dagligt sendt igennem vores offentlige IT platforme. Men langt størstedelen af vores IT infrastruktur er udenfor vores kontrol.
-
-Tag Danmarks love som et eksempel: de sørger for at staten og borgerne opfører sig ordentligt og de skaber tryghed og livskvalitet.
-Fordi vores IT systemer er lukkede kan vi ikke se om de overholder vores love.
-Vi kan faktisk ikke se om det er kvalitet eller skrammel.
-Hvorfor har vi ikke love, der sikrer at vores offentlige IT opfører sig ordentligt?
+dokumenter bliver dagligt sendt igennem vores offentlige IT platforme. Men langt størstedelen af vores IT infrastruktur er udenfor vores kontrol, da systemerne er lukkede. Dette bevirker, at koden bag systemerne ikke kan kvalitetssikres og det beværliggører kontrol med om koden overholder gældende love.
```

Comment:
Igen, kan bedre lide den eksisterende formulering. Det er mere spændende at læse.
Hvad er formålet med ændringen?

Reply:
Jeg kan bedre lide den nye version som en foreslagstekst, fordi den ikke stiller (omend retoriske) spørgsmål, og netop er mindre malerisk.

Jeg mener at foreslaget bør være klart, skarpt, og ikke for prosaisk. De spændende historier og formuleringer hører mere hjemme på hjemmesiden, synes jeg. Det kommer an på hvad vi ser formålene for de forskellige medier som. Jeg ser borgerforslaget som noget tættere-på-juridisk og mindre malerisk; den breder kommunikation hører hjemme på hjemmeside og i andre medier vi måtte række ud igennem.

Hvis jeg skal koge det ned lyder den tidligere version for holdningsfyldt, hvor den nye lyder mere som om den holder sig til fakta. "Er det her ikke dumt?" versus "Sådan ligger landet".

Reply:
Gør det lige ekstra klart: jeg synes ikke de tidligere formuleringer var dårlige. Bare at det officielle forslag nok ikke er der de hører hjemme.

Måske burde vi starte nogle andre dokumenter med flere formuleringer? I hvert fald efter vi er klar på kernen efter i morgen.